src/meta/src/stream/stream_graph.rs

```diff
@@ -99,83 +104,69 @@ type GlobalTableIdGen = GlobalIdGen<{ IdCategory::Table }>;
 type GlobalActorId = GlobalId<{ IdCategory::Actor }>;
 type GlobalActorIdGen = GlobalIdGen<{ IdCategory::Actor }>;
 
-/// A list of actors with order.
+/// A list of actor IDs.
 #[derive(Debug, Clone)]
-struct OrderedActorLink(pub Vec<GlobalActorId>);
+struct GlobalActorIds(pub Vec<GlobalActorId>);
 
-impl OrderedActorLink {
+impl GlobalActorIds {
     pub fn as_global_ids(&self) -> Vec<u32> {
         self.0.iter().map(|x| x.as_global_id()).collect()
     }
 }
 
-struct StreamActorDownstream {
```

Comment:
The `Downstream` is used for creating dispatchers for actors. Now we directly create the dispatcher without using this intermediate representation.